bundles/org.openhab.binding.sagercaster/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -163,6 +164,7 @@
 		<item-type>String</item-type>
 		<label>@text/trendLabel</label>
 		<description>@text/trendDescription</description>
+		<category>line</category>
```

Comment:
This category is not listed in documentation: https://www.openhab.org/docs/concepts/categories.html#channel-categories

Reply:
It exists and works : 
![image](https://user-images.githubusercontent.com/3328789/144715251-594746f8-aee6-4134-9bb6-6ed5d86a79c5.png)

Reply:
Camelcased.

Reply:
Yes, found here https://github.com/openhab/openhab-webui/blob/main/bundles/org.openhab.ui.iconset.classic/src/main/resources/icons/line.png
Documentation is outdated !